rules/windows/privilege_escalation_uac_bypass_event_viewer.toml

```diff
@@ -14,7 +14,67 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Bypass UAC via Event Viewer"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Bypass UAC via Event Viewer
+
+Windows User Account Control (UAC) allows a program to elevate its privileges (tracked as integrity levels ranging from
```

Comment:
Was this first sentence copied from an external source? I'd like to break it up and maybe re-organize it, but wanted to check whether can should be altered.

Reply:
Yep, https://attack.mitre.org/techniques/T1548/002/